ghcide/test/exe/DependentFileTest.hs

```diff
@@ -15,28 +14,32 @@ import           Language.LSP.Protocol.Types    hiding
                                                  SemanticTokensEdit (..),
                                                  mkRange)
 import           Language.LSP.Test
-import           Test.Hls.FileSystem            (FileSystem, toAbsFp)
-import           Test.Tasty
+import           Test.Hls
+
 
 tests :: TestTree
 tests = testGroup "addDependentFile"
-    [testGroup "file-changed" [testWithDummyPluginEmpty' "test" test]
+    [testGroup "file-changed" [testCase "test" $ runSessionWithTestConfig def
+        {testShiftRoot=True
+        , testDirLocation=Right (mkIdeTestFs [])
```

Comment:
I'll do an abstraction over this if I am seeing more occurence of this in other places when migrating tests. Not worth it now. Since it only occurs in one file now.